src/app/pages/MediaArticlePage/PagePromoSections/RelatedContentSection/index.styles.tsx

```diff
@@ -0,0 +1,41 @@
+import { css, Theme } from '@emotion/react';
+
+export default {
+  singleItemWrapper: ({ spacings }: Theme) =>
+    css({
+      width: '100%',
+      margin: `0 -${spacings.FULL}rem`,
+    }),
```

Comment:
Are we missing the media queries for the singleItemWrapper from here: https://github.com/bbc/simorgh/blob/c752bf0ee0283ab79b51f8267a6435baacba938d/src/app/pages/MediaArticlePage/PagePromoSections/RelatedContentSection/index.styles.jsx#L48-L57

Reply:
Also worth checking the others just in case 👍

Reply:
Great spot! Was just working on that.